tests/phpunit/tests/auth.php

```diff
@@ -100,6 +100,31 @@ public function test_password_trimming() {
 		}
 	}
 
+	/**
+	 * Test hooking into wp_set_password().
+	 *
+	 * @ticket 57436
+	 */
+	public function test_wp_set_password_action() {
+		$user_id             = self::$user_id;
+		$expected_meta_value = 'Meta value';
+
+		add_action(
+			'wp_set_password',
+			function( $password, $user_id ) {
+				update_user_meta( $user_id, 'my-password-user-meta', 'Meta value' );
+				clean_user_cache( $user_id );
+			},
+			10,
+			2
+		);
+
+		wp_set_password( 'A simple password', $user_id );
+		$user_meta_value = get_user_meta( $user_id, 'my-password-user-meta', true );
+
+		$this->assertSame( $expected_meta_value, $user_meta_value );
```

Comment:
Instead of assign `$expected_meta_value` variable we should pass value directly and remove that variable.

```suggestion
		$this->assertSame( 'Meta value', $user_meta_value );
```

Reply:
Thanks for the proposal, but it is now outdated since the test was entirely replaced with a MockAction() as per Sergey's suggestion :)